packages/web-app-admin-settings/src/index.ts

```diff
@@ -15,14 +17,20 @@ const appInfo = {
   isFileEditor: false
 }
 
-const routes = ({ $permissionManager }) => [
+const routes = ({ $ability }: { $ability: Ability }) => [
   {
     path: '/',
     redirect: () => {
-      if ($permissionManager.hasSystemManagement()) {
+      if ($ability.can('read-all', 'Setting')) {
         return { name: 'admin-settings-general' }
       }
-      return { name: 'admin-settings-spaces' }
+      if ($ability.can('read-all', 'Account')) {
+        return { name: 'admin-settings-users' }
+      }
+      if ($ability.can('read-all', 'Space')) {
+        return { name: 'admin-settings-spaces' }
+      }
+      throw Error('Insufficient permissions')
```

Comment:
isn't there also a `read-all` for `Groups`? 🤔 At least there was a `group-management` permission backend side at some point... if that's the case you could add the default redirect to the group management view as well.

Edit: yes, you're already using the `read-all` for `Groups` for displaying the `Groups` nav item. So it's missing here. :-)